doc/dev/issues/resolve_issues_effectively.md

```diff
@@ -11,6 +11,8 @@ Most issues can be classified into 3 categories. They are
 ## Usage error (For users)
 If you are not familiar with the SDK usage of a service, you can find relevant examples in [this repo][sample repo] in most cases.
 
+For some common errors, you can check [here](#Summary Of Common Errors).
```

Comment:
please check the view 👍 
![image](https://user-images.githubusercontent.com/70930885/171531277-c9e523eb-50ee-4414-9c1c-1b0ee03c2b5a.png)